Owncloud iOs Client/Tabs/FileTab/Share/ShareMainViewController.m

```diff
@@ -697,7 +742,16 @@ - (void) presentActivityViewForShareLink:(NSURL *)urlShareLink {
        UIActivityTypeSaveToCameraRoll,
        UIActivityTypePostToWeibo]];
     
-    [self presentViewController:activityView animated:YES completion:nil];
+    if (IS_IPHONE) {
```

Comment:
We should fine a way of doing this in a general way without repeating ```if (IS_IPHONE)```  once and again.

But not now. Let's take note and create an issue about technical debt.